### PR TITLE
Custom citizen names page

### PR DESCRIPTION
**Pull request by Ampaex**
It adds a new page in misc tab which explains how to change the citizen names. I also created a new folder for store every citizen names template.

### CODE REVIEW

File: source/misc/customcitizennames.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+For download the file click the _raw_ button on top of the page and then right-click -> save as...<br>
+These are the links we provide you for making your life a little bit easier:
+
+### Spanish
+
+[Download spanish template file](../source/misc/languageNameTemplates/spanishTemplate.toml)
+
+
+<br>
+**Don't forget it:** The file must have the name _minecolonies-server.toml_.
```
Comment (ravenbuilder934) [inline source/misc/customcitizennames.md:46]:
```suggestion
**Don't forget:** The file must have the name _minecolonies-server.toml_.
```

Comment (ravenbuilder934) [reply source/misc/customcitizennames.md:46]: Otherwise great!

**Comment by Ampaex** [comment]
I was testing the file, in spanish, shortened names are a little weird

File: source/misc/customcitizennames.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+ 
+---
+title: Minecolonies Wiki
+layout: recipes
+---
+# Custom citizen names
+
+Would you like having citizen names in your own language? This is your page!<br>
```
Comment (ravenbuilder934) [inline source/misc/customcitizennames.md:8]:
```suggestion
Would you like to have citizen names in your own language? Then this is your page!<br>
```

**Pull request merged**